sui/src/validator.rs

```diff
@@ -61,14 +63,18 @@ async fn main() -> Result<(), anyhow::Error> {
         address, net_cfg.host, net_cfg.port
```

Comment:
It would be great to modify this log line to log both listen address and expected public IP in a way that makes which is which clear.